docs/install/managing-multiple-projects.md

```diff
@@ -1,3 +1,103 @@
 # Managing Multiple Projects
 
-Documentation coming soon! See https://github.com/google/knative-gcp/issues/460
+A general Knative-GCP installation configures three identities (represents as Google service accounts) on your project.
+
+1. Identity for the Control Plane.
+2. Identity for the GCP Broker.
+3. Identity for the Source. (This is the default identity for the Source.
+Users can use this identity for multiple Source custom objects,
+or create additional identities in the form of Google service accounts for different Source custom objects,
+to achieve more fine-grained configurations)
+
+In general Knative-GCP installation guide, each Knative-GCP component consumes/creates Google Cloud resources within the scope of the project
+(the project where the kubernetes cluster contained your Knative-GCP components resides). For example, if your kubernetes cluster is in Project A, and
+you apply StorageSource in this kubernetes cluster, it will create a Google Cloud Storage Bucket, a Google Cloud PubSub Topic, and a Google Cloud PubSub Subscription,
+all belong to Project A.
```

Comment:
```suggestion
(the project where the kubernetes cluster containing your Knative-GCP components resides). For example, if your kubernetes cluster is in Project A, and
you apply [CloudStorageSource](../examples/cloudstoragesource) in this kubernetes cluster, then the Google Cloud Storage Notification, Google Cloud PubSub Topic, and Google Cloud PubSub Subscription will be created in Project A.
```

Reply:
Oh! Thanks for stopping me from making this ridiculous mistake for the Storage Source. I have no idea about why I wrote bucket not notification here :(